package.json

```diff
@@ -41,30 +42,20 @@
     "chai": "4.3.4",
     "eslint": "^7.25.0",
     "express": "4.17.1",
+    "jest": "^26.6.3",
     "mocha": "^8.3.2",
     "morgan": "1.10.0",
     "open": "^8.0.6",
+    "rimraf": "^3.0.2",
+    "rollup": "^2.45.2",
+    "rollup-plugin-commonjs": "^10.1.0",
+    "rollup-plugin-node-resolve": "^5.2.0",
     "sinon": "10.0.0",
+    "terser": "^5.6.1",
+    "typescript": "^4.2.4",
     "uglifyify": "^5.0.2"
```

Comment:
If I understand correctly, no longer needed
```suggestion
```

Reply:
Correct.  Thank you.

Reply:
Fixed